docs/metamacros.md

```diff
@@ -37,10 +37,38 @@ Example:
 """
 ```
 
+By default metamacros are 'nestable', which means that an author may write metamacros within
+metamacros. In some cases this may not be the behaviour that is desired. Nesting can be disabled on
+a per-definition basis by using the ``raw`` modifier:
+
+```
+metamacro(::META"name"raw, body, mod, obj) = ...
+```
+
+Example: nested metamacros.
+
+```julia
+"""
+
+!!summary(Set the value for a field in an object's metadata. !!set(url:https://github.com/MichaelHatherly/Docile.jl))
+
+
+!!setget(outer:Outer
+    !!setget(level1:Level1
+        !!setget(level2:Level2
+            !!setget(level3:Level3
+                !!setget(level4:Level4
+                    !!setget(level5:Level5
+                        !!setget(level6:Level6)))))))
```

Comment:
A little too much nesting here. I think just one deep is fine as an illustration.